### ISSUE TITLE
KeyError when converting cbc results to csv when storage is included

### ISSUE THREAD

**Issue by HauHe** [OSeMOSYS/otoole#60]
Hi, 
I tried to convert the cbc results that were created by running a model with the fast code to csv results. I get the below KeyError.
I'm not sure if this is related to the fact that the model includes storage components or something else.

```python
c:\nobackup\haukeh\programs\anaconda3\envs\otoole\lib\site-packages\pandas\core\frame.py:3509: SettingWithCopyWarning:
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  self[k1] = value[k2]
Error in sys.excepthook:
Traceback (most recent call last):
  File "c:\nobackup\haukeh\programs\anaconda3\envs\otoole\lib\site-packages\otoole\cli.py", line 294, in exception_handler
    print("{}: {}".format(exception_type.__name__, exception.message))
AttributeError: 'KeyError' object has no attribute 'message'

Original exception was:
Traceback (most recent call last):
  File "c:\nobackup\haukeh\programs\anaconda3\envs\otoole\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\nobackup\haukeh\programs\anaconda3\envs\otoole\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Nobackup\haukeh\programs\anaconda3\envs\otoole\Scripts\otoole.exe\__main__.py", line 7, in <module>
  File "c:\nobackup\haukeh\programs\anaconda3\envs\otoole\lib\site-packages\otoole\cli.py", line 299, in main
    args.func(args)
  File "c:\nobackup\haukeh\programs\anaconda3\envs\otoole\lib\site-packages\otoole\cli.py", line 102, in result_matrix
    convert_cbc_to_csv(args.from_path, args.to_path)
  File "c:\nobackup\haukeh\programs\anaconda3\envs\otoole\lib\site-packages\otoole\results\convert.py", line 289, in convert_cbc_to_csv
    csv = convert_dataframe_to_csv(df, input_data)
  File "c:\nobackup\haukeh\programs\anaconda3\envs\otoole\lib\site-packages\otoole\results\convert.py", line 202, in convert_dataframe_to_csv
    types = {index: sets[index]["dtype"] for index in indices}
  File "c:\nobackup\haukeh\programs\anaconda3\envs\otoole\lib\site-packages\otoole\results\convert.py", line 202, in <dictcomp>
    types = {index: sets[index]["dtype"] for index in indices}
KeyError: 's in STORAGE' 
```

**Comment by willu47** [comment]
Thanks for this @HauHe - that's not a very helpful error message is it!  I'll see if I can fix that as a starter.  Could you share the cbc results you were trying to convert so that I can pin point the issue?

**Comment by HauHe** [comment]
Nope it isn't. 
Thanks for the help! 
[DE_baseline_sol.txt](https://github.com/OSeMOSYS/otoole/files/5209182/DE_baseline_sol.txt)
Here the cbc file.

**Comment by willu47** [comment]
This is now fixed in v0.8.1

**Comment by HauHe** [comment]
Thanks a lot!
